Authlete/Api/AuthleteApiV3.cs

```diff
@@ -0,0 +1,1279 @@
+        const string CLIENT_GET_LIST_API_PATH               = "/api/client/get/list";
+        const string CLIENT_SECRET_REFRESH_API_PATH         = "/api/{0}/client/secret/refresh/{1}";
+        const string CLIENT_SECRET_UPDATE_API_PATH          = "/api/{0}/client/secret/update/{1}";
+        const string CLIENT_UPDATE_API_PATH                 = "/api/{0}/client/updat/{1}";
+        const string REQUESTABLE_SCOPES_DELETE_API_PATH     = "/api/{0}/client/extension/requestable_scopes/delete/{1}";
+        const string REQUESTABLE_SCOPES_GET_API_PATH        = "/api/{0}/client/extension/requestable_scopes/get/{1}";
+        const string REQUESTABLE_SCOPES_UPDATE_API_PATH     = "/api/{0}/client/extension/requestable_scopes/update/{1}";
+        const string GRANTED_SCOPES_GET_API_PATH            = "/api/{0}/client/granted_scopes/get/{1}";
+        const string GRANTED_SCOPES_DELETE_API_PATH         = "/api/{0}/client/granted_scopes/delete/{1}";
+        const string CLIENT_AUTHORIZATION_DELETE_API_PATH   = "/api/{0}/client/authorization/delete/{1}";
+        const string CLIENT_AUTHORIZATION_GET_LIST_API_PATH = "/api/{0}/client/authorization/get/list";
+        const string CLIENT_AUTHORIZATION_UPDATE_API_PATH   = "/api/{0}/client/authorization/update/{1}";
+        const string BC_AUTHENTICATION_API_PATH             = "/api/{0}/backchannel/authentication";
+        const string BC_AUTHENTICATION_COMPLETE_API_PATH    = "/api/{0}/backchannel/authentication/complete";
+        const string BC_AUTHENTICATION_FAIL_API_PATH        = "/api/{0}/backchannel/authentication/fail";
+        const string BC_AUTHENTICATION_ISSUE_API_PATH       = "/api/{0}/backchannel/authentication/issue";
+        const string DEVICE_AUTHORIZATION_API_PATH          = "/api/{0}/device/authorization";
+        const string DEVICE_COMPLETE_API_PATH               = "/api/{0}/device/complete";
+        const string DEVICE_VERIFICATION_API_PATH           = "/api/{0}/device/verification";
+        const string PUSHED_AUTH_REQ_API_PATH               = "/api/{0}/pushed_auth_req";
+        
+        
+        private readonly string _mAuth;
+        private readonly long? _mServiceId;
+        private JsonWebKey _mDpopJwk;
+
+
+        /// <summary>
+        /// Constructor with configuration.
+        /// </summary>
+        ///
+        /// <param name="configuration">
+        /// Configuration to access an Authlete API server.
+        /// </param>
+        public AuthleteApiV3(IAuthleteConfiguration configuration)
+        {
+             
+            if (configuration == null)
+            {
+                throw new ArgumentNullException(nameof(configuration));
+            }
+            
+            BaseUri                 = CreateBaseUri(configuration);
+            ExtractDpop(configuration);
+            Settings                = new SettingsImpl(this);
+            
+            // Parse the Authlete API version specified by the configuration
```

Comment:
Add a period at the end of the sentence. The same thing applies to the other lines.